### PR TITLE
@ashfurrow => Route to /auction/ when tapping on a bid button

### PR DESCRIPTION
**Pull request by orta**
Fixes: https://github.com/artsy/eigen/issues/529

### CODE REVIEW

**Comment by ashfurrow** [comment]
Sorry I missed this – it needs a rebase.

**Comment by orta** [comment]
done'd

File: Artsy.xcodeproj/xcshareddata/xcschemes/Artsy.xcscheme [hunk 1]
```diff
@@ -14,8 +14,8 @@
             buildForAnalyzing = "YES">
             <BuildableReference
                BuildableIdentifier = "primary"
-               BlueprintIdentifier = "7B6F3179CDD265E55DF25EFE"
-               BuildableName = "libPods.a"
+               BlueprintIdentifier = "7B290E9A1822F602465768A6"
+               BuildableName = "Pods.framework"
```
Comment (alloy) [inline Artsy.xcodeproj/xcshareddata/xcschemes/Artsy.xcscheme]: Where does this framework reference come from?

Comment (orta) [inline Artsy.xcodeproj/xcshareddata/xcschemes/Artsy.xcscheme]: oh, sneakyyyyyyy, must have come in from me working on a frameworks branch

**Comment by orta** [comment]
Removed the sneaky `Pods.framework` ref

File: Artsy.xcodeproj/xcshareddata/xcschemes/Artsy.xcscheme (hunk 1 above)
Comment (alloy) [inline Artsy.xcodeproj/xcshareddata/xcschemes/Artsy.xcscheme]: Ok ok, so I’m not going crazy, at least not _that_ crazy :smile:

File: Artsy/Classes/Utils/ARSwitchBoard.m [hunk 2]
```diff
@@ -182,7 +182,7 @@ - (ARArtworkSetViewController *)loadArtworkSet:(NSArray *)artworkSet inFair:(Fai
 
 - (UIViewController *)loadBidUIForArtwork:(NSString *)artworkID inSale:(NSString *)saleID
 {
-    NSString *path = [NSString stringWithFormat:@"/feature/%@/bid/%@", saleID, artworkID];
+    NSString *path = [NSString stringWithFormat:@"/auctions/%@/bid/%@", saleID, artworkID];
```
Comment (orta) [inline Artsy/Classes/Utils/ARSwitchBoard.m]: this needs changing to `/auction/` - artsy doesn't do the rails-y routing :-/

Comment (ashfurrow) [inline Artsy/Classes/Utils/ARSwitchBoard.m]: Gotcha. Let me know @alloy if you want me to fix that.

Comment (alloy) [inline Artsy/Classes/Utils/ARSwitchBoard.m]: @ashfurrow Did done: #558.

Comment (ashfurrow) [inline Artsy/Classes/Utils/ARSwitchBoard.m]: :tada:

**Pull request merged**